test_common/harness/imageHelpers.cpp

```diff
@@ -796,7 +796,8 @@ void get_max_sizes(
 
     (*numberOfSizes) = 0;
 
-    if (image_type == CL_MEM_OBJECT_IMAGE1D)
+    if (image_type == CL_MEM_OBJECT_IMAGE1D
+        || image_type == CL_MEM_OBJECT_IMAGE1D_BUFFER)
     {
 
         size_t M = maximum_sizes[0];
```

Comment:
M should be adjusted to be within the max_pixels calculated allow to account for max allocation size.
For other object types, max_pixels factors into the math for other dimensions (after the max is locked) but there's no adjustment for the 1D cases.

Reply:
Done